use/2016-01-26-SNP-selection.Rmd

```diff
@@ -211,19 +211,25 @@ ggplot(loc2temp, aes(x = factor(loc7), y = temp)) +
  ylab("Temperature (Centigrade)")
 
 mod <- glm(cbind(loc7, 2 - loc7) ~ temp, family = binomial)
-summary(mod) ## This loci is not significantly correlated to temperature
+summary(mod) ## This locus is not significantly correlated to temperature
  
 ```
 
- 
+Now that we are done with the analysis, we can remove the temporary file that
+*PCAdapt* has created:
```

Comment:
It doesn't seem like the fact that *PCAdapt* creates temporary file(s) (and under which name) is introduced somewhere earlier. Shouldn't it, so that this is coming less out of the blue.

Also, it seems from the `unlink()` call that this temp file is created in the working directory, which is problematic because it has consequences for the user (such as, the working directory needs to be writable by the user, and needs to have sufficient space available) that for example are not necessarily met on HPC environments. So isn't it worth alerting the user (reader) to that early on?